README.md

```diff
@@ -133,9 +133,27 @@ make test
 
 ### Compiling CX on Windows
 
-Compiling CX on Windows requires a recent version of Git to be installed. 
+Compiling CX on Windows requires a recent version of Git to be installed.   
```

Comment:
I would shorten this whole bit to something like: 

```
Compiling CX on Windows requires Git and Pacman to be installed. 

In order to install Pacman, visit [MYSYS2](example.xyz) <-- put the actual link to the download section here
and download the latest version of MYSYS2. After installing pacman, chose `Run MYSYS2 now`. 

Update the package databse and base packages with:

pacman -Syu

Reply:
Can we install Git with pacman?

Reply:
Both `mingw` as well as `git` can be installed with `pacman` so we should probably do it like that.

Reply:
Yes